docs/specs/openapi.yaml

```diff
@@ -70,15 +154,120 @@ paths:
+    get:
+      summary: Get service logs
+      tags:
+        - services
+      description: |
+        Fetch previously-written logs from the given services,
+        or request logs from the given services and follows them, in JSON lines format.
+      parameters:
+        - name: services
+          in: query
+          description: |
+            Service name to filter logs by.
+
+            To get logs from multiple services, specify this parameter multiple times.
+
+            If not set, returns logs for all services.
+          schema:
+            type: string
+        - name: follow
+          in: query
+          description: |
+            Whether to follow the logs (keep streaming new logs).
+
+            Must be "true" or "false"; defaults to false.
+          schema:
+            type: string
+            enum: ["true", "false"]
+            default: "false"
+        - name: n
+          in: query
+          description: |
+            Number of log entries to retrieve.
+
+            If `follow` is true, this parameter is ignored and defaults to zero.
+
+            In non-follow mode, the default value is server-defined, currently 30.
+
+            If `n` is -1, all available logs are returned (up to a server-defined limit).
+
+            If `n` is 0 and `follow` is false, a server-defined default number of logs is returned.
+
+            If `n` is a positive integer, up to that many logs are returned.
+          schema:
+            type: integer
+            format: int32
+      responses:
+        "200":
+          description: Get service logs in JSON lines format.
+          content:
+            application/x-ndjson:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/logs"
+              example:
+                {
+                  "time": "2024-12-27T02:32:53.185Z",
+                  "service": "svc1",
+                  "message": " * Serving Flask app 'main'"
+                }
+  /v1/signals:
+    post:
+      summary: Send a signal to services
+      tags:
+        - services
+      description: Send a signal to each of the specified services.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                signal:
+                  type: string
+                  description: The signal to send. For example, "SIGHUP", "SIGINT", "SIGKILL".
+                services:
+                  type: array
+                  description: List of service names to send the signal to.
+                  items:
+                    type: string
+                  minItems: 1  # At least one service is required.
+              required:
+                - signal
+                - services
+              example:
+                {
+                  "signal": "SIGHUP",
+                  "services": ["svc1"]
+                }
```

Comment:
I think this can be indented up one level (to be the same level as `schema`). Currently, Swagger UI is showing the example under "Schema" as well as under "Example Value". We only need the example to show under "Example Value"

Reply:
Thanks for pointing this out, this is for sure a human error, it should only show under "example value".